.github/workflows/rust.yml

```diff
@@ -10,20 +10,18 @@ jobs:
   build:
     strategy:
       matrix:
-        os: ["ubuntu-latest"]
-        rust_channel: ["stable", "beta", "nightly", "1.42.0"]
+        os: ['ubuntu-latest']
+        rust_channel: ['stable', 'beta', 'nightly', '1.42.0']
```

Comment:
Why do the different quotes matter?

Reply:
They don't - this was just for consistency and it is in its own commit to clarify the intent.

Reply:
This is really only a cosmetic change. Consistency isn't a great reason here. There are at least three different contexts for quotes in this file: YAML strings, github actions expressions, and shell commands, and in all cases there are reasons why you might need different quotes.

The downside of this sort of change is that it is noise in the git history, and takes time to review. I'll keep it now that it has been done though.